detections/endpoint/create_remote_thread_in_shell_application.yml

```diff
@@ -0,0 +1,64 @@
+name: Create Remote Thread In Shell Application
+id: 10399c1e-f51e-11eb-b920-acde48001122
+version: 1
+date: '2021-08-04'
+author: Teoderick Contreras, Splunk
+type: batch
+datamodel:
+- Endpoint
+description: This search is to detect suspicious process injection in command shell.
+  This technique was seen in IcedID where it execute cmd.exe process to inject its
+  shellcode as part of its execution as banking trojan. It is really uncommon to have
+  a create remote thread execution in the following application.
+search: '`sysmon` EventCode=8 TargetImage IN ("*\\cmd.exe", "*\\powershell*") | stats
+  count min(_time) as firstTime max(_time) as lastTime by  TargetImage TargetProcessId
+  SourceProcessId  EventCode StartAddress SourceImage Computer | `security_content_ctime(firstTime)`
+  | `security_content_ctime(lastTime)` | `create_remote_thread_in_shell_application_filter`'
+how_to_implement: To successfully implement this search, you need to be ingesting
+  logs with the process name, parent process, and command-line executions from your
+  endpoints. If you are using Sysmon, you must have at least version 6.0.4 of the
+  Sysmon TA.
+known_false_positives: unknown
+references:
+- https://thedfirreport.com/2021/07/19/icedid-and-cobalt-strike-vs-antivirus/
+tags:
+  analytic_story:
+  - Icedid
+  dataset:
+  - https://media.githubusercontent.com/media/splunk/attack_data/master/datasets/malware/icedid/simulated_icedid/windows-sysmon.log
+  kill_chain_phases:
+  - Exploitation
+  mitre_attack_id:
+  - T1055
+  product:
+  - Splunk Enterprise
+  - Splunk Enterprise Security
+  - Splunk Cloud
+  required_fields:
+  - _time
+  - SourceImage
+  - TargetImage
+  - TargetProcessId
+  - SourceProcessId
+  - StartAddress
+  - EventCode
+  - Computer
+  security_domain: endpoint
+  impact: 70
+  confidence: 100
+  risk_score: 70
+  context:
+  - source: endpoint
```

Comment:
can you change this to `Source:Endpoint` , caps on first letter and no space after :

Reply:
thanks @patel-bhavin. i already modify all context for those new detection with that format. including the Stage: